recipes/ghostpii/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "ghostPii" %}
+{% set version = "1.0.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/ghostPii-{{ version }}.tar.gz
+  sha256: f67c0b9150a55e3b14cbfd4bec5ece848a13a74d79ba247a6dcad0969e3f24f9
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - IPython
```

Comment:
```suggestion
```
This package is not listed in requirements upstream.

Reply:
Apparently IPython is imported, [but not listed in the requirements upstream](https://github.com/capnion/ghostpii_client/blob/010bf3a6c2e61719ee5e13f2b776c3f13b27625e/setup.cfg#L22). This is a bug that should be fixed upstream.